### PR TITLE
#5

### PR DESCRIPTION
**Pull request by Viviaann**
Frequency of Meetings

### CODE REVIEW

**Comment by Viviaann** [comment]
two different versions for frequency of meetings

File: CONSTITUTION.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ Will there be a required number of meetings each quarter? Each year?
 Will minutes be taken and/or required for official meetings? 
 NOTE: Minutes can be useful for disseminating information to members who can’t make it, as well as for officially recording organization procedure and decision-making.
 ```
+There will be a quarterly general body meeting for all members, biweekly meetings for board members, and no required meetings for general members. General members will only need to attend meetings for the project/workshop that they signed up for. Meeting minutes will be taken for general body meetings and all board meetings.
```
Comment (asg017) [inline CONSTITUTION.md]:
```suggestion
There will be a quarterly general body meeting for all members, biweekly meetings for board members, and no required meetings for general members. Meeting minutes will be taken for general body meetings and all board meetings. 
```

**Review by asg017 — COMMENTED** [review]
one sentence change!

**Comment by asg017** [comment]
5 👍 !

**Pull request merged**